copylot/assemblies/photom/gui/windows.py

```diff
@@ -879,6 +1027,27 @@ def _move_marker_and_update_sliders(self):
             self.photom_controls.mirror_widgets[
                 self.photom_controls._current_mirror_idx
             ].mirror_y_slider.setValue(new_coords[1][0])
+    
+    def _roi_tracing(self):
+        if self.selected_shape_id != None:
+            shape = self.shapes[self.selected_shape_id]
+            new_pattern_points = []
+            if shape.pattern_points:
+                for position in shape.pattern_points:
```

Comment:
currently these will be fed without a delay. Let's add a pattern_delay variable that we can set to pause between positions.

Reply:
the default can be zero.